pkg/workspace_template.go

```diff
@@ -698,6 +699,9 @@ func (c *Client) generateWorkspaceTemplateWorkflowTemplate(workspaceTemplate *Wo
 		return nil, err
 	}
 
+	workflowTemplateManifest = strings.NewReplacer(
```

Comment:
This looks good. 

Any reason for using this instead of `strings.ReplaceAll`?

Reply:
> Any reason for using this instead of `strings.ReplaceAll`?

Good question, based on the following references:

https://pkg.go.dev/github.com/metakeule/replacer?tab=doc#pkg-overview
https://medium.com/@vikram.ingawale91/golang-strings-replace-vs-strings-replacer-a7b2d2b71593

It seems that it's more memory efficient and generally faster than the alternatives.

Reply:
Sounds good to me!